inst/app/modules/main_map.R

```diff
@@ -0,0 +1,51 @@
+
+hk_accidents_valid <- dplyr::filter(hk_accidents, !is.na(latitude) & !is.na(longitude))
+
+# Leaflet default expect WGS84 (crs 4326), need custom CRS for HK1980 Grid (crs 2326)
+# https://rstudio.github.io/leaflet/projections.html
+hk_accidents_valid_sf <- st_as_sf(hk_accidents_valid, coords = c("longitude", "latitude"), crs = 4326, remove = FALSE)
+
+# Need to convert to POSIXct again, otherwise reactive filtering does not work
+# TODO: investigate why
+hk_accidents_valid_sf$Date <- as.Date(hk_accidents_valid_sf$Date, format = "%Y-%m-%d")
+
+CARTODB_POSITRON_TILE_URL <- "https://cartodb-basemaps-{s}.global.ssl.fastly.net/light_all/{z}/{x}/{y}.png"
+
+output$main_map <- renderLeaflet({
+  leaflet() %>%
+    addTiles(urlTemplate = CARTODB_POSITRON_TILE_URL) %>%
+    setView(114.2, 22.3, zoom = 12)
```

Comment:
Preference is to always name the arguments unless it is super obvious. Long and lat are especially obtuse so better to name these arguments.

```suggestion
    setView(lng = 114.2, lat = 22.3, zoom = 12)
```